mne/io/kit/tests/test_kit.py

```diff
@@ -137,10 +140,34 @@ def test_ch_loc():
 
     # test when more than one marker file provided
     mrks = [mrk_path, mrk2_path, mrk3_path]
-    read_raw_kit(sqd_path, mrks, elp_path, hsp_path, preload=False)
+    read_raw_kit(sqd_path, mrks, elp_txt_path, hsp_txt_path, preload=False)
     # this dataset does not have the equivalent set of points :(
     raw_bin.info['dig'] = raw_bin.info['dig'][:8]
     raw_py.info['dig'] = raw_py.info['dig'][:8]
     assert_dig_allclose(raw_py.info, raw_bin.info)
 
+
+def test_hsp_elp():
+    """Test KIT usage of *.elp and *.hsp files against *.txt files
+    """
+    raw = read_raw_kit(sqd_path, mrk_path, elp_txt_path, hsp_txt_path)
+    raw_leg = read_raw_kit(sqd_path, mrk_path, elp_path, hsp_path)
+
+    trans = raw.info['dev_head_t']['trans']
+    trans_leg = raw_leg.info['dev_head_t']['trans']
+    assert_array_almost_equal(trans, trans_leg, decimal=5)
```

Comment:
I think you might be right about the origin of the inaccuracy, the *.elp only contains 5 meaningful digits to begin. At UMD we use the pen digitizer which directly exports *.seq, *.hsp and *.elp files (even though it's also produced by Polhemus). I just added a test for the final transformed fiducials, they are also equal to about 0.1 mm, which is probably more accurate than the whole digitization procedure is to begin with...